Spring-server/src/main/java/ikpi63holding/postclient/ImapConnector.java

```diff
@@ -9,13 +10,8 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Properties;
-import javax.mail.Flags;
-import javax.mail.Folder;
-import javax.mail.Message;
-import javax.mail.MessagingException;
-import javax.mail.PasswordAuthentication;
-import javax.mail.Session;
-import javax.mail.Store;
+import javax.mail.*;
```

Comment:
I think it'll be better if we use more "java.mail.Something" instead of "java.mail.*"